src/client/components/Headline/Headline.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
```

Comment:
Ah, cool with the typing 👌 I just copied from some other Storybook files since I haven't really worked much with Storybook (for some reason I keep on forgetting that we use that).

Sure, I can make another story as well, thought it was fine to just change the background but maybe it's easier to have two stories :)